src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthModule.cs

```diff
@@ -313,7 +316,7 @@ public ResultWrapper<byte[]> eth_sign(Address addressData, byte[] message)
         public Task<ResultWrapper<Keccak>> eth_sendTransaction(TransactionForRpc transactionForRpc)
         {
             Transaction tx = transactionForRpc.ToTransactionWithDefaults();
-            return SendTx(tx);
+            return SendTx(tx, TxHandlingOptions.ManagedNonce);
```

Comment:
If nonce is non 0, then will it change it?

Reply:
yeah, I was thinking about it, what do you think?